### PR TITLE
Extend tests to check explicit storage class name

### PR DESCRIPTION
**Pull request by brybacki**
**What this PR does / why we need it**:

Storage class name is always set on the PVC when that PVC is created
by the CDI. Updated a few tests to check for this behavior. Added two new scenarios.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/datavolume_test.go [hunk 1]
```diff
@@ -1681,6 +1681,73 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			By("Restore the profile")
 			updateStorageProfileSpec(f.CrClient, defaultScName, *originalProfileSpec)
 		})
+		It("[test_id:8170]Import succeeds when storage class is not specified, but access mode is", func() {
+			defaultScName := utils.DefaultStorageClass.GetName()
+
+			By(fmt.Sprintf("configure storage profile %s", defaultScName))
```
Comment (awels) [inline tests/datavolume_test.go:1687]: I would put the piece of code that saves the storage profile in a BeforeEach, and the piece that restores it in an AfterEach

Comment (brybacki) [reply tests/datavolume_test.go:1689]: done

**Comment by brybacki** [comment]
As the imageio was fixed, the test should now pass.

**Pull request merged**